scripts/mockGuardianSpy.ts

```diff
@@ -0,0 +1,347 @@
+import {
+    Server,
+    ServerCredentials,
+} from '@grpc/grpc-js';
+import {MockGuardians} from "@certusone/wormhole-sdk/lib/cjs/mock";
+const GUARDIAN_PRIVATE_KEY = '0x5756c5018454b83225f8e8264651c3a8032c4ac931e1d310d81239754f1bb791';
+const guardians = new MockGuardians(0, [
+    GUARDIAN_PRIVATE_KEY,
+  ]);
+
+const RetryProvider = require("../packages/package-utils").RetryProvider;
+// Random key
+
+const vaa = "AQAAAAABADgbykvOsEtlw6QJf5oKPXacqMKYVAvlmhCPVzROoatsM7CMKOpPuaGT59ztB4i4yjAmE5ytzdqSQKXHflUh8N0AZhgDQ2ZmAQAABQAAAAAAAAAAAAAAADd9VaeSjARuGO67YZd+cU0qdkcqAAAAAAAAmfQPAgAAAAAAAAAAAAAAACk4+usSFQG1YMCJvYu8nF/QlM9eAAUSSFQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABIZWxsb1Rva2VuAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=="
+
+import { EmitterFilter, FilterEntry, SpyRPCServiceService, SubscribeSignedVAAResponse} from '../lib/wormhole/sdk/js-proto-node/src/spy/v1/spy'
+
+
+const ethers = require("ethers");
+
+// eslint-disable-next-line import/no-unresolved
+const bridgeAbi = require("../artifacts/contracts/testHelpers/WormholeMock.sol/WormholeMock.json").abi;
+// eslint-disable-next-line import/no-unresolved
+const wormholeBridgeForColonyAbi = require("../artifacts/contracts/bridging/WormholeBridgeForColony.sol/WormholeBridgeForColony.json").abi;
+
+function ethereumAddressToWormholeAddress(address: string) {
```

Comment:
I wouldn't be opposed to calling this `e2w` for brevity but I have a feeling you won't like that

Reply:
Your feeling is correct 😆 

I'll see if I can come up with something else, but open to other suggestions. Some things just have long names though, and brevity should not come at the cost of intelligibility.